verified_model/Site/North/epsg_code.json

```diff
@@ -0,0 +1,74 @@
+{
+    "value": 32628,
```

Comment:
I did not check the actual value. Do you plan to have a comparison later to the current values as part of the validation step?

Reply:
I didn't think about it, but you are correct that we should introduce a comparison / validation step of the values in the DB and those in this repository. That should be a relative simple routine, which simply loops over all parameters and does a comparison. I've opened an issue to not forget it (issue #12).

Reply:
Perfect, thanks!